esterni/piano-di-qualifica/components/B-valutazioni-per-il-miglioramento.tex

```diff
@@ -81,5 +83,47 @@ \subsubsection{Errori nei documenti}
 Gli errori riguardanti le norme sono stati commessi soprattutto nelle prime fasi di stesura dei documenti e sono stati subito corretti nella prima parte di verifica per ogni documento quando tutto il gruppo ha appreso meglio le norme interne imposte.
 Per i periodi troppo verbosi è stato utile l'utilizzo dell'Indice di Gulpease, ma soprattutto una buona comunicazione all'interno del gruppo per rimediare a frasi troppo lunghe.
 
+\subsection{Secondo periodo (RP)}
+Vengono esposti i problemi che sono sorti nella seconda parte di lavoro valutando di conseguenza quali soluzioni adottare in merito.
+I principali problemi riscontrati riguardano:
+\begin{itemize}
+    \item \textbf{L'organizzazione}: problemi inerenti all'organizzazione interna del gruppo e nella gestione del lavoro;
+    \item \textbf{La correzione}: problemi relativi a incomprensioni nella gestione delle correzioni dei prodotti presentati nella Revisione dei Requisiti;
+    \item \textbf{Gli strumenti di lavoro}: problemi relativi alle scelte sugli strumenti utilizzati dal gruppo in questo periodo.
+\end{itemize}
+La corretta rilevazione di questi errori e le relative soluzioni adottate sono state facilitate dall'esperienza del gruppo maturata nella prima parte del progetto e grazie soprattutto alla disponibilità e alla comunicazione fra i diversi componenti del team.
+Di seguito vengono affrontati nello specifico i problemi riscontrati.
+
+\subsubsection{Valutazioni sull’organizzazione}
+\label{sub:valutazioni_organizzazione}
+Vengono definiti i problemi sull'organizzazione del team, punto fondamentale per la continuazione dello sviluppo del progetto.
+
+\paragraph{Incomprensioni sui requisiti}
+\label{sub:incomprensioni_sui_requisiti}
+Durante questo periodo all'interno del gruppo sono sorti dei dubbi sulla corretta comprensione delle richieste del committente, queste incomprensioni sono sorte durante la realizzazione del Proof of concept il quale ci ha permesso di addentrarci meglio nel problema richiesto.
```

Comment:
Durante questo periodo all'interno del gruppo sono sorti dei dubbi sulla corretta comprensione delle richieste del committente, queste incomprensioni sono nate durante la realizzazione del Proof of concept, il quale ci ha permesso di addentrarci meglio nel problema richiesto.